sidecar/main.go

```diff
@@ -73,7 +79,7 @@ func main() {
 
 	sidecarServer := server.NewSidecarServer(*controllerEndpoint)
 	sidecarServer.RegisterService(service.NewIdentityServer(csiClient.Client))
-	sidecarServer.RegisterService(service.NewReclaimSpaceServer(csiClient.Client, kubeClient))
+	sidecarServer.RegisterService(service.NewReclaimSpaceServer(csiClient.Client, kubeClient, stagingPath))
```

Comment:
you can pass `*stagingPath` so that you do not need to care about the pointer indirection in the `ReclaimSpaceServer` struct